src/main/kotlin/racing/Game.kt

```diff
@@ -0,0 +1,25 @@
+package racing
+
+class Game {
+    val carList: MutableList<Car> = mutableListOf()
+    private val gameEngine = GameEngine
+
+    fun run(carNumber: Int, turnNumber: Int) {
+        initializeCarList(carNumber)
+        ResultView().showReadyMessage()
+
+        (0 until turnNumber).forEach { _ ->
```

Comment:
```suggestion
        repeat(turnNumber) {
```
kotlin [repeat](https://kotlinlang.org/api/latest/jvm/stdlib/kotlin/repeat.html) API를 익혀보고 사용해보시면 어떨까요? (이하 동일합니다)

Reply:
반영하였습니다! 유용한 API 소개해주셔서 감사합니다!